index.html

```diff
@@ -24,15 +24,45 @@ <h1 class="title">Hi! I am Ezra nice to meet you.</h1>
                 If you like what you see and have a project you need coder, don't 
                 husitate to contact me.
             </p>
-            <h3 class="links">LET'S CONNECT</h3>
+        </main>
+    </section>
+    <section class="work-page">
+        <img src="images/works/portfolio1.svg" alt="work1" width="100%">
+        <h2 class="work-title">Tonic</h2>
+        <ul class="clients">
+            <li>CANOPY</li>
+            <img src="images/works/Counter.svg" alt="counter" class="counter">
+            <li>BackEndDev</li>
+            <img src="images/works/Counter.svg" alt="counter" class="counter">
+            <li>2015</li>
+        </ul>
+        <p class="highlights">
+            A daily selection of privately personalized reads
+            no accounts for sign-ups required.
+        </p>
+        <ul class="tags">
+            <li><button>HTML</button></li>
+            <li><button>CSS</button></li>
+            <li><button>Js</button></li>
+        </ul>
+        <button class="project-detail">See <br> Project</button>

```

Comment:
- Kindly work on the padding of your `see project `  button to match the Figma design

![image](https://user-images.githubusercontent.com/98453979/198716227-cc4257e9-7e7f-4e31-8aa0-bb62903f9825.png)

### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/198716847-39c63cfc-3080-4924-a2b6-298bce284b80.png)